### PR TITLE
Spotify click event

### PR DESCRIPTION
**Pull request by Nollain**
Add click events to Spotify module to support : 
- PlayPause
- Next song
- Previous Song

Those are configured in the i3status.conf file

### CODE REVIEW

**Comment by gfalcone** [comment]
2 in a row, stop the guy

**Comment by tobes** [comment]
@Nollain did you try the wonderful `mpris` module?

File: py3status/modules/spotify.py [hunk 1]
```diff
@@ -27,12 +27,23 @@
...
+
 i3status.conf example:
 
 ```
 spotify {
     format = "{title} by {artist} -> {time}"
     format_down = "no Spotify"
+    next_song = 4
+    previous_song = 5
+    play_paused = 1
```
Comment (tobes) [inline py3status/modules/spotify.py]: by convention in modules we use `button_next`, `button_play` etc

Comment (Nollain) [reply py3status/modules/spotify.py]: Oups I didn't see that, my bad, i made the changes

File: py3status/modules/spotify.py [hunk 2]
```diff
@@ -27,12 +27,23 @@
     color_paused: Song is stopped or paused, defaults to color_degraded
     color_playing: Song is playing, defaults to color_good
 
+Click Events:
+    You can add some config to your i3status.conf to  create click events
+    For example
```
Comment (lasers) [inline py3status/modules/spotify.py]: * We can get rid of this section in favor of `Configuration parameters:` docs for the buttons.
* This PR is missing the buttons for the `Configuration parameters:` docs.
* This PR is missing the button variables in the `available configuration parameters` section. 
* See `cmus` / `moc` if you want to do some quick copy and paste for the docs.

File: py3status/modules/spotify.py (hunk 1 above)
Comment (lasers) [reply py3status/modules/spotify.py]: (Suggestion): Use `button_pause` instead and default this with `1` too.

File: py3status/modules/spotify.py [hunk 3]
```diff
@@ -171,6 +191,31 @@ def spotify(self):
...
+            - Pause/Play
+            - Next song
+            - Previous song
+
+        Sleep to "invalidate cache" and give the time to change the song to refresh the display
+        """
+
+        button = event['button']
+        if button is None:
+            pass
```
Comment (lasers) [inline py3status/modules/spotify.py]: I'd get rid of the docstring above and the `if button is None` too as this does not avoid anything.

**Comment by ultrabug** [comment]
Thank you @Nollain, will do a quick QA on this after merging.

**Pull request merged**